database/dbBookStore.js

```diff
@@ -0,0 +1,40 @@
+const pgp = require( 'pg-promise' )()
+const connectionString = process.env.DATABASE_URL || 'postgress://localhost:5432/guarded-porcupine-db'
+const db = pgp( connectionString )
+
+const getAllBooks = () =>
+  db.any( `SELECT * FROM bookshb ORDER BY title` )
+
+const getBook = (id) =>
+    db.one( `SELECT * FROM bookshb where id=$1`, [id] )
+
+const addBook = ({ title, author, description, genre, image_url, price, keywords }) =>
+  db.oneOrNone(
+    `INSERT INTO
+       (title, author, description, genre, image_url, price, keywords)
+    VALUES
+      ($1, $2, $3, $4, $5, $6, $7)`,
+    [title, author, description, genre, image_url, price, keywords] )
+
+const updateBook = (id, book) => {
```

Comment:
You can remove the curly braces and return here and do an implicit return, to keep it consistent with the style of the rest of these functions.